src/couch_replicator/src/couch_replicator_job.erl

```diff
@@ -299,6 +348,12 @@ handle_info({'EXIT', Pid, normal}, #rep_state{workers = Workers} = State) ->
         Workers ->
             %% Processes might be linked by replicator's auth plugins so
             %% we tolerate them exiting `normal` here and don't crash
+            ?LOG_WARNING(#{
+                what => unknown_process_exit,
```

Comment:
Ambiguous meaning (see another similar comment earlier).

Reply:
Changed to `linked_process_exit`